cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    describe('Emails edit modals' + runInfo(), () => {
+      it('add additional emails and display those as "unverified" with default privacy', () => {
+        // Expect changes to be display outside and inside of the modal
+      })
+      it('remove emails', () => {
```

Comment:
Should we show an alert message right after an email delete button is pressed.

Reply:
I'm inclined to say 'No' to this as it just seems like overkill. Are secondary emails really _so_ important that a user needs to be reminded they've just clicked the button to delete one?

If we keep things consistent behaviour-wise then as soon as a user clicks the delete button on a secondary email address it'll be removed from the list. It literally disappears! However, it's not actually deleted until the user confirms the action by clicking 'Save changes'. Cancelling by clicking outside of the modal, clicking the [X] button or pressing 'Cancel' will undo these changes.

Reply:
Yes!, I don't think we need to ask the users to confirm the email deletion if now the email deleting will only happen after the save button is pressed.  

I will keep this comment as `unresolved` for now until we have these scenarios well defined.

Reply:
As we have concluded at previous meetings, we will not show this deleted email message.